polaris/hub/polarisfs.py

```diff
@@ -120,3 +128,71 @@ def cat_file(
         with fsspec.open(signed_url, "rb", **kwargs) as f:
             data = f.read()
         return data[start:end]
+
+    def rm(self, path: str, recursive: bool = False, maxdepth: Optional[int] = None):
```

Comment:
For the documentation GH CICD, you will need to specify a return type. 

```suggestion
    def rm(self, path: str, recursive: bool = False, maxdepth: Optional[int] = None) -> None:
```